app/models/waste_exemptions_engine/transient_address.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require WasteExemptionsEngine::Engine.root.join("app", "models", "waste_exemptions_engine", "transient_address")
+
+module WasteExemptionsEngine
+  class TransientAddress
+    include WasteExemptionsEngine::CanBeSearchedLikeAddress
```

Comment:
I don't think it needs the Module name

```suggestion
    include CanBeSearchedLikeAddress
```

Again I could be wrong 😀